nix/holochain/default.nix

```diff
@@ -3,9 +3,9 @@ let
   script = pkgs.writeShellScriptBin "acorn-release"
   ''
   set -euxo pipefail
+  ./clean.sh
   ./update-dna-version.sh
   ./update-ui-version.sh
-  ./clean.sh
```

Comment:
if we don't run an aspect of `clean.sh` after running the `update-x` commands, then we end up with much larger .app files, because all of `acorn-ui` and `acorn-hc` repos are bundled in